includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -136,14 +136,22 @@ public function get_description() {
 	 * @return bool
 	 */
 	public function is_enabled_at_checkout( $order_id = null ) {
+		// Check capabilities first.
+		if ( ! $this->is_capability_active() ) {
+			return false;
+		}
```

Comment:
This check needs to be first, before the subscriptions check: otherwise if the cart contains a subscription product and the payment method is reusable, we will return `true`, even though the capability may be inactive.

Reply:
Credit to @brettshumaker for spotting this one. 👍

Reply:
Wow, good find @brettshumaker! This was easy to miss!

Reply:
Thanks!